### PR TITLE
Update intro text

### PR DESCRIPTION
**Pull request by psatyal**
Changelog:

- Updated introduction to use language closer to the app description 
- Updated titles of links to the documentation (so that they use the same titles as in the documentation)

Signed-off-by: Parimal Satyal <88370597+psatyal@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+	
+On the Nextcloud end, it allows users to:
 
-Please report issues and bugs in: https://community.openproject.org/projects/nextcloud-integration/work_packages
+* Link files and folders with work packages in OpenProject
+* Find all work packages linked to a file or a folder
+* View OpenProject notifications via the dashboard
+* Search for work packages using Nextcloud's search bar
+
+On the OpenProject end, users are able to:
```
Comment (wielinde) [inline README.md:12]: @psatyal Is there an emtpy line missing?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,24 @@
-# OpenProject integration into Nextcloud
+# OpenProject Integration
 
-OpenProject integration provides the possibility to link OpenProject work packages to a file in NextCloud, a dashboard
-widget for displaying OpenProject notifications, a search provider to search for work packages through the unified
-search and can display the count of OpenProject notifications as NextCloud notification.
+This application enables integration between Nextcloud and open-source project management software OpenProject. 
+
```
Comment (wielinde) [inline README.md:4]: @psatyal the space characters in this line are not necessary (bad practise to keep them in, as other might delete it and potentially create merge conflicts later).

**Review by wielinde — CHANGES_REQUESTED** [review]
Thanks @psatyal, please find my comments about formatting. Text wise they everything looks great.

File: README.md (hunk 2 above)
Comment (psatyal) [reply README.md]: Good catch, will remove.

**Review by psatyal — COMMENTED** [review]
Thank you @wielinde. Made the two changes.

**Review by wielinde — APPROVED** [review]
@psatyal Thank you! Looks good.

**Pull request merged**